packages/metrics/src/config/routes.ts

```diff
@@ -525,6 +526,15 @@ export const getRoutes = () => {
         },
         tags: ['api']
       }
+    },
+    // new Audit handler
+    {
+      method: 'POST',
+      path: '/new/audit/event',
```

Comment:
What do you think about having this route named quite strictly with the REST naming convention, so something like `POST /audit/events` ?

```suggestion
      path: '/audit/events',
```